lerna.json

```diff
@@ -3,10 +3,9 @@
   "concurrency": 8,
   "ci": false,
   "npmClient": "npm",
-  "npmClientArgs": [
-    "--no-package-lock"
-  ],
+  "npmClientArgs": ["--no-package-lock"],
   "packages": [
+    "dev/*",
```

Comment:
side note: we should consider moving test-studio and others to 'dev/' as well, it's a bit of a misnomer to call them "examples" :)

Reply:
Agreed! We can do that after this is merged?